src/adapter/bridges/OFTBridge.ts

```diff
@@ -0,0 +1,266 @@
+import { Contract, ethers, Signer } from "ethers";
+import { BridgeTransactionDetails, BaseBridgeAdapter, BridgeEvents } from "./BaseBridgeAdapter";
+import {
+  BigNumber,
+  EventSearchConfig,
+  Provider,
+  TOKEN_SYMBOLS_MAP,
+  isDefined,
+  paginatedEventQuery,
+  isContractDeployedToAddress,
+  assert,
+  EvmAddress,
+  Address,
+  resolveToken,
+} from "../../utils";
+import { processEvent } from "../utils";
+import { CHAIN_IDs, PUBLIC_NETWORKS } from "@across-protocol/constants";
+import { CONTRACT_ADDRESSES, IOFT_ABI_FULL } from "../../common";
+
+import {
+  IOFT,
+  MessagingFeeStruct,
+  SendParamStruct,
+} from "@across-protocol/contracts/dist/typechain/contracts/interfaces/IOFT";
+
+import { utils } from "@across-protocol/sdk";
+const { toBytes32 } = utils;
```

Comment:
`toBytes32()` is re-exported out of `src/utils/SDKUtils.ts` :+1: 

```suggestion
import {
  BigNumber,
  EventSearchConfig,
  Provider,
  TOKEN_SYMBOLS_MAP,
  isDefined,
  paginatedEventQuery,
  isContractDeployedToAddress,
  assert,
  EvmAddress,
  Address,
  resolveToken,
  toBytes32,
} from "../../utils";
import { processEvent } from "../utils";
import { CHAIN_IDs, PUBLIC_NETWORKS } from "@across-protocol/constants";
import { CONTRACT_ADDRESSES, IOFT_ABI_FULL } from "../../common";

import {
  IOFT,
  MessagingFeeStruct,
  SendParamStruct,
} from "@across-protocol/contracts/dist/typechain/contracts/interfaces/IOFT";
```

Reply:
fixed [41dbeb8](https://github.com/across-protocol/relayer/pull/2133/commits/41dbeb8bd086e8947107fa3c23301f1d32eb1a59)